### PR TITLE
Added intl_countries, intl_languages, intl_locales twig extensions

### LINKED ISSUE #8: Get list of countries
Add a twig function to load all available countries:

```php
    public function getFunctions()
    {
        return [
            new \Twig_SimpleFunction('intl_countries', [$this, 'getCountries']),
        ];
    }

    public function getCountries(string $displayLocale): array
    {
        return Intl::getRegionBundle()->getCountryNames($displayLocale);
    }
```

### PR DESCRIPTION
**Pull request by alexander-schranz**
```twig
{% for country in intl_countries() %}
    {{ country }}
{% endfor %}
```

```twig
{% for language in intl_languages() %}
    {{ language }}
{% endfor %}
```

```twig
{% for locale in intl_locales() %}
    {{ locale }}
{% endfor %}
```

fixes #8

### CODE REVIEW

**Comment by alexander-schranz** [comment]
ping @massiveart/lead-developer

**Comment by dominikmatt** [comment]
👍

File: tests/IntlTwigExtensionTest.php [hunk 1]
```diff
@@ -86,4 +122,27 @@ public function testLocale()
             $this->intlTwigExtension->getLocale('de_AT', 'en')
         );
     }
+
+    public function testLocalse()
```
Comment (c00n84) [inline tests/IntlTwigExtensionTest.php]: @alexander-schranz `testLocales`

**Comment by c00n84** [comment]
else ACK

File: tests/IntlTwigExtensionTest.php (hunk 1 above)
Comment (alexander-schranz) [reply tests/IntlTwigExtensionTest.php]: fixed

**Pull request merged**